miniapp/miniapp_bt_band_to_tridiag.cpp

```diff
@@ -153,27 +167,23 @@ struct BacktransformBandToTridiagMiniapp {
 
       // print benchmark results
       if (0 == world.rank() && run_index >= 0) {
-        std::cout << "[" << run_index << "]"
-                  << " " << elapsed_time << "s"
-                  << " " << gigaflops << "GFlop/s"
-                  << " " << dlaf::internal::FormatShort{opts.type} << " " << mat_e_host.size() << " "
-                  << mat_e_host.blockSize() << " " << opts.b << " " << comm_grid.size() << " "
-                  << pika::get_os_thread_count() << " " << backend << std::endl;
+        std::cout << "[" << run_index << "]" << " " << elapsed_time << "s" << " " << gigaflops
+                  << "GFlop/s" << " " << dlaf::internal::FormatShort{opts.type} << " "
+                  << mat_e_host.size() << " " << mat_e_host.blockSize() << " " << opts.b << " "
+                  << comm_grid.size() << " " << pika::get_os_thread_count() << " " << backend << " ("
+                  << 0l << ", " << opts.last_eval_idx << ")" << std::endl;
         if (opts.csv_output) {
           // CSV formatted output with column names that can be read by pandas to simplify
           // post-processing CSVData{-version}, value_0, title_0, value_1, title_1
-          std::cout << "CSVData-2, "
-                    << "run, " << run_index << ", "
-                    << "time, " << elapsed_time << ", "
-                    << "GFlops, " << gigaflops << ", "
-                    << "type, " << dlaf::internal::FormatShort{opts.type}.value << ", "
-                    << "matrixsize, " << mat_e_host.size().rows() << ", "
-                    << "blocksize, " << mat_e_host.blockSize().rows() << ", "
-                    << "band_size, " << opts.b << ", "
-                    << "comm_rows, " << comm_grid.size().rows() << ", "
-                    << "comm_cols, " << comm_grid.size().cols() << ", "
-                    << "threads, " << pika::get_os_thread_count() << ", "
-                    << "backend, " << backend << ", " << opts.info << std::endl;
+          std::cout << "CSVData-2, " << "run, " << run_index << ", " << "time, " << elapsed_time << ", "
+                    << "GFlops, " << gigaflops << ", " << "type, "
+                    << dlaf::internal::FormatShort{opts.type}.value << ", " << "matrixsize, "
+                    << mat_e_host.size().rows() << ", " << "blocksize, " << mat_e_host.blockSize().rows()
+                    << ", " << "band_size, " << opts.b << ", " << "comm_rows, "
+                    << comm_grid.size().rows() << ", " << "comm_cols, " << comm_grid.size().cols()
+                    << ", " << "threads, " << pika::get_os_thread_count() << ", " << "backend, "
+                    << backend << ", " << opts.info << ", " << "first eigenvalue index, " << 0l << ", "
+                    << "last eigenvalue index, " << opts.last_eval_idx << std::endl;
```

Comment:
I think you are using the bugged clang-format available in ubuntu 24.04

Reply:
I updated to `18.1.8` as in CI.